src/Symfony/Component/Lock/Store/StoreFactory.php

```diff
@@ -60,6 +67,9 @@ public static function createStore(#[\SensitiveParameter] object|string $connect
             case 'semaphore' === $connection:
                 return new SemaphoreStore();
 
+            case str_starts_with($connection, 'dynamodb://'):
+                return new DynamoDbStore($connection);
```

Comment:
This class check is also necessary here.
```suggestion
                self::requireBridgeClass(DynamoDbStore::class, 'symfony/amazon-dynamodb-lock');

                return new DynamoDbStore($connection);
```

Reply:
Done